### PR TITLE
Local

### PR DESCRIPTION
**Pull request by giannispan2103**

### CODE REVIEW

File: demo.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+if __name__ == "__main__":
+    listener = GlobalStreamListener(lan="en")
+    executor = StreamExecutor(listener)
+    executor.loop()
+
+# geopy.exc.GeocoderServiceError: [Errno -2] Name or service not known
+# geopy.exc.GeocoderTimedOut: Service timed out
+# ssl.SSLError: [SSL: KRB5_S_TKT_NYV] unexpected eof while reading (_ssl.c:2309)
+
```
Comment (ipavlopoulos) [inline demo.py]: Are these important?

File: web.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from flask import Flask
+from twitter.stream import loop
+app = Flask(__name__)
+
```
Comment (ipavlopoulos) [inline web.py]: What is the purpose of this flask? Will it serve to show the tweets in a map?

**Review by ipavlopoulos — APPROVED** [review]

File: demo.py (hunk 1 above)
Comment (ipavlopoulos) [reply demo.py]: It's hard to follow without comments, could you add some?

Comment (giannispan2103) [reply demo.py]: no

Comment (giannispan2103) [reply demo.py]: I ll remove them now

File: web.py (hunk 2 above)
Comment (giannispan2103) [reply web.py]: I was thinking to build a basic flask application but I thing this is an is for another pull requests, so I ll remove it

File: demo.py (hunk 1 above)
Comment (giannispan2103) [reply demo.py]: > It's hard to follow without comments, could you add some?

I suppose you mean at datahandler.py, so I did now

Comment (ipavlopoulos) [reply demo.py]: Good stuff! Thanks. I am downloading it now to see it in action.

File: web.py (hunk 2 above)
Comment (ipavlopoulos) [reply web.py]: Cool, ty!
